diff -crB SFML.orig/src/SFML/System/CMakeLists.txt SFML/src/SFML/System/CMakeLists.txt
*** SFML.orig/src/SFML/System/CMakeLists.txt	2011-12-26 12:31:48.558862874 +0100
--- SFML/src/SFML/System/CMakeLists.txt	2011-12-26 12:32:26.130860981 +0100
***************
*** 22,27 ****
--- 22,29 ----
      ${INCROOT}/Thread.inl
      ${SRCROOT}/ThreadLocal.cpp
      ${INCROOT}/ThreadLocal.hpp
+     ${SRCROOT}/D/ThreadImpl.cpp
+     ${SRCROOT}/D/ThreadImpl.hpp
      ${INCROOT}/ThreadLocalPtr.hpp
      ${INCROOT}/ThreadLocalPtr.inl
      ${INCROOT}/Utf.hpp
***************
*** 42,49 ****
          ${SRCROOT}/Win32/MutexImpl.hpp
          ${SRCROOT}/Win32/SleepImpl.cpp
          ${SRCROOT}/Win32/SleepImpl.hpp
-         ${SRCROOT}/Win32/ThreadImpl.cpp
-         ${SRCROOT}/Win32/ThreadImpl.hpp
          ${SRCROOT}/Win32/ThreadLocalImpl.cpp
          ${SRCROOT}/Win32/ThreadLocalImpl.hpp
      )
--- 44,49 ----
***************
*** 56,63 ****
          ${SRCROOT}/Unix/MutexImpl.hpp
          ${SRCROOT}/Unix/SleepImpl.cpp
          ${SRCROOT}/Unix/SleepImpl.hpp
-         ${SRCROOT}/Unix/ThreadImpl.cpp
-         ${SRCROOT}/Unix/ThreadImpl.hpp
          ${SRCROOT}/Unix/ThreadLocalImpl.cpp
          ${SRCROOT}/Unix/ThreadLocalImpl.hpp
      )
--- 56,61 ----
diff -crB SFML.orig/src/SFML/System/Thread.cpp SFML/src/SFML/System/Thread.cpp
*** SFML.orig/src/SFML/System/Thread.cpp	2011-12-26 12:31:48.558862874 +0100
--- SFML/src/SFML/System/Thread.cpp	2011-12-26 12:33:36.338857442 +0100
***************
*** 28,38 ****
  #include <SFML/System/Thread.hpp>
  
  
! #if defined(SFML_SYSTEM_WINDOWS)
!     #include <SFML/System/Win32/ThreadImpl.hpp>
! #else
!     #include <SFML/System/Unix/ThreadImpl.hpp>
! #endif
  
  
  namespace sf
--- 28,34 ----
  #include <SFML/System/Thread.hpp>
  
  
! #include <SFML/System/D/ThreadImpl.hpp>
  
  
  namespace sf
